lib/index.js

```diff
@@ -53,23 +38,30 @@ function MqttProtocol () {
       * "responseTopic": should be subscribe-only, so no one but this plugin can write in it
      */
     this.server.authorizePublish = function(client, topic, payload, callback) {
-      if (config.allowPubSub) {
-        callback(null, topic !== config.responseTopic);
+      if (this.config.allowPubSub) {
+        const isAllowed = topic !== this.config.responseTopic
+          && topic.indexOf('#') === -1
```

Comment:
`#` & `+` are mqtt wilcards. If we were allowing them, it would be possible to subscribe to `Kuzzle/#` for instance and get all requests from all clients.